src/browser/Terminal.ts

```diff
@@ -224,7 +224,7 @@ export class Terminal extends CoreTerminal implements ITerminal {
             const channels = color.toColorRGB(acc === 'ansi'
               ? this._colorManager.colors.ansi[req.index]
               : this._colorManager.colors[acc]);
-            this.coreService.triggerDataEvent(`${C0.ESC}]${ident};${toRgbString(channels)}${C0.BEL}`);
+            this.coreService.triggerDataEvent(`${C0.ESC}]${ident};${toRgbString(channels)}ESC \9c`);
```

Comment:
Think this is right:

```suggestion
            this.coreService.triggerDataEvent(`${C0.ESC}]${ident};${toRgbString(channels)}${C1.ST}`);
```

Reply:
Hmm, maybe I don't know how to use ESC notation 🤔

Reply:
Ah with `\` I meant the backslash ASCII char, which is at position 0x5c (or `\x5c` as hex-char). So ST would become `{C0.ESC}\x5c`, or `{C0.ESC}\\` (javascript needs double \\ to print one backslash). Maybe it is a good idea to create another convenient mapping for the escaped C1 chars in `common/data/EscapeSequences`, like this:
```typescript
const C1_escaped = {
  ST: `{C0.ESC}\\`,
  // add wotever other C1 is needed in escaped form
}
```
Then you can use it directly as `{C1_escaped.ST}` and so on.

The escaped form of C1 chars of interest can be formalized as: 
```typescript
function escape_char(c) {
  return '\x1b' + String.fromCharCode(c.charCodeAt(0) - 0x40)
}
```